components/Staking/StakingStepper.tsx

```diff
@@ -328,38 +335,61 @@ export const StakingStepper = ({ profile }: { profile: XProfile | null }) => {
     }
   }, [profile]);
 
+  const { data: isWalletUpdateRequired, isLoading } = useReadContract({
```

Comment:
@kupermind this is the part you can verify. please ignore [ZeroAddress as Address] in args, it won't happen (prevented by logic in query.enabled) and just added for type safeness